core/src/main/resources/hudson/diagnosis/MemoryUsageMonitor/index.jelly

```diff
@@ -61,7 +61,7 @@ THE SOFTWARE.
           </j:otherwise>
         </j:choose>
       </div>
-      <img src="graph?type=${type}&amp;width=500&amp;height=300" />
+      <img src="graph?type=${type}&amp;width=500&amp;height=300" alt="memory usage monitor graph" />
```

Comment:
```suggestion
      <img src="graph?type=${type}&amp;width=500&amp;height=300" alt="${%memory usage monitor graph}" />
```

The same applies to all `.jelly` files. (Previously mentioned in https://github.com/jenkinsci/jenkins/pull/9296#discussion_r1609285596)

Reply:
Fixed in 904cb3e4ac582523f2db3d70c1404cad5f8facb7